src/bandersnatch/mirror.py

```diff
@@ -1012,12 +1009,13 @@ async def mirror(
     mirror_url = config.get("mirror", "master")
     timeout = config.getfloat("mirror", "timeout")
     global_timeout = config.getfloat("mirror", "global-timeout", fallback=None)
+    proxy = config.get("mirror", "proxy", fallback=None)
```

Comment:
Hmm, are there docs about this configuration option?

Reply:
I've added docs for this part